scripts/run.sh

```diff
@@ -9,7 +9,8 @@ if [ ! -f $DATA_DIR/postgresql.conf ]; then
     chown postgres:postgres /data
 
     sudo -u postgres /usr/lib/postgresql/9.3/bin/initdb -E utf8 --locale en_US.UTF-8 -D $DATA_DIR
-    sed -i -e"s/^#listen_addresses =.*$/listen_addresses = '*'/" $DATA_DIR/postgresql.conf
+ echo  "shared_preload_libraries='pg_stat_statements'">> $DATA_DIR/postgresql.conf   
+ sed -i -e"s/^#listen_addresses =.*$/listen_addresses = '*'/" $DATA_DIR/postgresql.conf
```

Comment:
Yes you are right.
I only need to add this

echo  "shared_preload_libraries='pg_stat_statements'">>
$DATA_DIR/postgresql.conf

Can you please adjust this.

Thanks

Shashank


On Sat, Jul 25, 2015 at 3:44 AM, Ferran Rodenas <notifications@github.com>
wrote:

> In scripts/run.sh
> <https://github.com/frodenas/docker-postgresql/pull/2#discussion_r35470906>
> :
>
> > @@ -9,7 +9,8 @@ if [ ! -f $DATA_DIR/postgresql.conf ]; then
> >      chown postgres:postgres /data
> >
> >      sudo -u postgres /usr/lib/postgresql/9.3/bin/initdb -E utf8 --locale en_US.UTF-8 -D $DATA_DIR
> > -    sed -i -e"s/^#listen_addresses =.*$/listen_addresses = '*'/" $DATA_DIR/postgresql.conf
> > + echo  "shared_preload_libraries='pg_stat_statements'">> $DATA_DIR/postgresql.conf
> > + sed -i -e"s/^#listen_addresses =.*$/listen_addresses = '*'/" $DATA_DIR/postgresql.conf
>
> Is there any specific reason why you are doing this? By default it is
> already set to:
>
> listen_addresses = '*'
>
> So there is no need to do this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/frodenas/docker-postgresql/pull/2/files#r35470906>.
>